entry.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# switch storage driver to overlay2
+units="/lib/systemd/system/balena.service"
+if [ -f /etc/systemd/system/balena.service.d/balena.conf ]; then
+	units="${units} /etc/systemd/system/balena.service.d/balena.conf"
+fi
+for unit in $units; do
+	sed -i "s/-s aufs/-s overlay2/g" "${unit}"
```

Comment:
this can just put its own drop-in unit under `/run/` once https://github.com/balena-os/meta-balena/pull/2175 goes in